### PR TITLE
uploader package with updated peakinvestigator_saas.py and progress module

### PR DESCRIPTION
**Pull request by jct197**
from peakinvestigator import PeakInvestigatorSaaS
from peakinvestigator.actions import *
from peakinvestigator.progress import *

service = PeakInvestigatorSaaS()
action = UploadAction("VERSION", "USERID", "PASSWORD","PROJECT_ID","DATA_NAME")
response = service.execute(action)
action.process_response(response)
progress_factory = LoggerProgressPer()
service.upload(action,"FILE_NAME",progress_factory)

### CODE REVIEW

File: peakinvestigator/progress/logger.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+# of the BSD 3-Clause license.
+from peakinvestigator.progress.progress import *
+import logging
+import time
+import sys
+import os
+
+logging.info('Progress')
+
+class LoggerProg(Progress):
```
Comment (TDI-tenderholt) [inline peakinvestigator/progress/logger.py]: Don't abbreviate name. Use `LoggerProgress` instead.

File: peakinvestigator/progress/loggerpercentage.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+# of the BSD 3-Clause license.
+from peakinvestigator.progress.progress import *
+import logging
+import time
+import sys
+import os
+
+logging.getLogger('Progress')
+
+class LoggerProgPer(Progress):
```
Comment (TDI-tenderholt) [inline peakinvestigator/progress/loggerpercentage.py]: Don't abbreviate name. Use `LoggerPercentProgress` instead.

**Review by TDI-tenderholt — CHANGES_REQUESTED** [review]
Very good! Just two nitpicks about abbreviating class names. Fix those, and I think this can be merged.

File: peakinvestigator/progress/loggerpercentage.py (hunk 2 above)
Comment (jct197) [reply peakinvestigator/progress/loggerpercentage.py]: Both changes are added in latest commit.

**Pull request merged**